rest_framework_tus/urls.py

```diff
@@ -8,5 +8,5 @@
 router.register(r'files', UploadViewSet, basename='upload')
 
 urlpatterns = [
-    url(r'', include((router.urls, 'rest_framework_tus'), namespace='api'))
+    re_path(r'', include((router.urls, 'rest_framework_tus'), namespace='api'))
```

Comment:
This should also work (I think);
```suggestion
    path('', include((router.urls, 'rest_framework_tus'), namespace='api'))
```